graphql/schemabuilder/pagination.go

```diff
@@ -281,9 +299,19 @@ func getConnection(key string, nodes []interface{}, args ConnectionArgs) (Connec
 		startCursor = edges[0].Cursor
 	}
 
+	if returnsConnInfo {
+		connInfo := out[1].Interface().(PaginationInfo)
+		pageInfo := PageInfo{
+			HasNextPage: connInfo.HasNextPage,
+			HasPrevPage: connInfo.HasPrevPage,
+			StartCursor: startCursor,
+			EndCursor:   endCursor,
+		}
+		return Connection{TotalCount: connInfo.TotalCount(), Edges: edges, PageInfo: pageInfo}, nil
+	}
 	pageInfo := PageInfo{HasNextPage: nextPage, EndCursor: endCursor, StartCursor: startCursor, HasPrevPage: prevPage, Pages: pages}
```

Comment:
can we block out `pages` in the passthrough case?

Reply:
Yes